packages/@ember/-internals/routing/lib/system/route.js

```diff
@@ -1352,15 +1341,13 @@ let Route = EmberObject.extend(ActionHandler, Evented, {
     // Assign the route's controller so that it can more easily be
     // referenced in action handlers. Side effects. Side effects everywhere.
     if (!this.controller) {
-      let propNames = getWithDefault(this, '_qp.propertyNames', []);
```

Comment:
`_qp.propertyNames` is always array https://github.com/emberjs/ember.js/blob/master/packages/@ember/-internals/routing/lib/system/route.js#L213